index.html

```diff
@@ -7,10 +7,56 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css?family=Anonymous+Pro|PT+Serif&display=swap');
+    </style>
+    <style>
+        @import url('https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300&display=swap');
+    </style>
+
+
+
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <nav id="navigation">
```

Comment:
Think about using the `<header>` element that was covered in last week's lesson - the nav would semantically make sense to live there in this case.